### PR TITLE
Fix backdrop

### PR DESCRIPTION
**Pull request by Hadegda**
## Description

Return to the previous version of backdrop (without depth writing)
Set renderOrder for backdrop to 0 and to1 for other materials for render backdrop earlier than other materials

## Type of changes
- Bug fix (non-breaking change which fixes an issue)

## Checklist
- [x] I have read [CONTRIBUTING](https://github.com/epam/miew/blob/master/CONTRIBUTING.md) and [CODE_OF_CONDUCT](https://github.com/epam/miew/blob/master/CODE_OF_CONDUCT.md) guides.
- [x] I have followed the code style of this project.
- [x] I have run `yarn run ci`: lint and tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation / The changes do not need docs update.

### CODE REVIEW

File: src/gfx/materials.js [hunk 1]
```diff
@@ -94,13 +90,11 @@ const materialList = [
       depthWrite: true,
       transparent: true,
       toonShading: false,
-      side: THREE.DoubleSide,
     },
   }, {
     id: 'GL',
     name: 'Glass',
     shortName: 'Glass',
-    depthWrite: true,
```
Comment (NataliaDSmirnova) [inline src/gfx/materials.js]: Why have you removed the depthWrite property from the material?

File: src/gfx/gfxutils.js [hunk 2]
```diff
@@ -436,6 +436,16 @@ function processMaterialForShadow(root, material) {
   });
 }
 
+function processGeoForSetRenderOrder(root, idMaterial) {
```
Comment (NataliaDSmirnova) [inline src/gfx/gfxutils.js]: I want to suggest simplification for the function name. What do you think about setGeoRenderOrder?

**Comment by NataliaDSmirnova** [comment]
Oh, thank you for fixing our Backdrop. See my comments for the changes required!

File: src/gfx/materials.js (hunk 1 above)
Comment (Hadegda) [reply src/gfx/materials.js]: In Glass material "depthWrite: true" was written twice/ So I removed one of them (which was written in the wrong place - depthWrite usually defined in "values" section)

File: src/gfx/gfxutils.js (hunk 2 above)
Comment (Hadegda) [reply src/gfx/gfxutils.js]: Rename function to processObjRenderOrder

**Comment by NataliaDSmirnova** [comment]
Thank you one more time!

**Pull request merged**